connectors/acoustic/acoustic/model/reporting_raw_recipient_data_export.py

```diff
@@ -0,0 +1,43 @@
+"""
+Class describing the structure of raw recipient data record/row
+"""
+
+import attr
+
+
+# TODO: maybe just use dataclass? Is the initial DWH stage enforcing schema?
+#       What should we do here?
```

Comment:
The connector gets the schema on read, deduplicating by whatever you put as the primary key